addons/website/static/src/js/editor/snippets.editor.js

```diff
@@ -27,6 +32,35 @@ weSnippetEditor.SnippetsMenu.include({
         ['website-settings', _lt("Website Settings")],
     ],
 
+    /**
+     * @override
+     */
+    start() {
+        this._super(...arguments);
+        this.ownerDocument = this.$el[0].ownerDocument;
+        this.$currentAnimatedText = $();
+        const $animateTextButton = $(`
+            <div title="${_t('Animate text')}"
+                class="btn fa fa-play fa-fw o_we_animate_text">
+            </div>
```

Comment:
It is ok this does nothing when I click on this button when the selection spans over bold and non bold for example... but only because I know how this works. There should be something to explain to the user why that does not work (unless @pierre-pde disagrees). Maybe a disabled button with a tooltip explanation or a notification on click... check with PDE.

Reply:
As @pierre-pde  is busy today and is not able to answer before the end of the day. I have already added a notification to inform the user that the selection cannot be animated. ("The current text selection cannot be animated.")
I let you decide if we keep this ;)

Reply:
Hi @bvr-odoo , sorry for the delay, I think we need to help user to understand how he can have animation to work.
So, instead of "The current text selection cannot be animated", I would recommend "The current text selection cannot be animated.  Try clearing the format and try again."